recipes/sentry-native/all/conanfile.py

```diff
@@ -119,7 +119,7 @@ def requirements(self):
                 self.requires("breakpad/cci.20210521")
         if self.options.qt:
             self.requires("qt/5.15.2")
-            self.requires("openssl/1.1.1k")
+            self.requires("openssl/1.1.1l")
```

Comment:
I'm curious. Does 3.0.0 work?

Reply:
It seems to work locally but I didn't test further.

Reply:
Thanks. I was wondering about the API compatibility.

Reply:
Compatibility is really freaking close, someone contribute to a project I am apart of https://github.com/Thalhammer/jwt-cpp 

One broken const https://github.com/Thalhammer/jwt-cpp/commit/17fdd26d97dec5b77c03ccef9057b343894e54dd